### ISSUE TITLE
path_prefix option is not used when belongs_to association is set.

### ISSUE THREAD

**Issue by yhsgw** [remiprev/her#518]
I am setting the path_prefix option as follows:

```
Her::API.setup url: ENV['API_HOST'] do |c|
  # Request
  c.use Faraday::Request::UrlEncoded
  
  # Response
  c.use Her::Middleware::DefaultParseJSON
  
  # Adapter
  c.use Faraday::Adapter::NetHttp

  c.path_prefix = '/api/v1/'
end

class User
  include Her::Model
  belongs_to :organization
end

class Organization
  include Her::Model
end
```

path_prefix option is not used when belongs_to association is set.
```
> user = User.find(1) 
# GET “api/v1/users/1", response is:
# {
#   "id": 1,
#   "name": “yhsgw”,
#   "organization_id": 2
# }

> user.organization # goes to wrong path (path_prefix being stripped)
# GET "/organizations/2”
```

I would like to request to “/api/v1/organizations/2”.

belongs_to association is building path that “/#{name.to_s.pluralize}/:id", 
however, when having the / at the beginning of the path will tell Faraday that it's an absolute path.
(https://github.com/lostisland/faraday/issues/823)

**Comment by j1mmie** [comment]
Hoping to see a fix for this as well. Tried to make a PR myself but got lost in the meta-programming 🙃

**Comment by j1mmie** [comment]
I see this is fixed in #514. I've commented there to get a status update

**Comment by noctivityinc** [comment]
Any idea when this is going to be merged?

**Comment by zacharywelch** [comment]
Fixed by #514 and released in [v1.1.0](https://github.com/remiprev/her/releases/tag/v1.1.0). Thanks!
